_data/education.yml

```diff
@@ -397,6 +397,20 @@ websites:
     twitter: sqanews
     facebook: ScottishQualificationsAuthority
     img: mysqa.png
+    
+  - name: North Carolina State University
+    url: https://ncsu.edu
+    img: ncsu.png
+    tfa:
+      - sms
+      - phone
+      - hardware
```

Comment:
Maybe I didn't spot it, but which hardware 2FA method is used here (other than U2F keys)?
```suggestion
```

Reply:
Sounds good, thanks for all the helpful comments.